tests/fast-integration/skr-test/provision-skr.js

```diff
@@ -30,13 +30,13 @@ describe('Provision SKR instance', function() {
   });
 
   after('Print Shoot Info', async function() {
-    if (shootInfo != undefined) {
-      console.log(options);
+    // print data out for spinnaker
+    if (options && options.instanceID) {
+      console.log(`SPINNAKER_PROPERTY_INSTANCE_ID=${options.instanceID}`);
+    }
 
-      // print data out for spinnaker
-      console.log('***Print out data for spinnaker***');
+    if (shootInfo) {
       console.log(`SPINNAKER_PROPERTY_PROVISIONED=true`);
```

Comment:
```suggestion
      console.log(`PROPERTY_PROVISIONED=true`);
```

Reply:
It needs to have the prefix "SPINNAKER_PROPERTY_" as required by spinnaker.

Reply:
could you elaborate on it why?

Reply:
It used in spinnaker provision stage to pass data to next stage. More info: https://spinnaker.io/docs/guides/user/kubernetes-v2/run-job-manifest/#spinnaker_property_